CharlieBackend.Panel/HtmlHelpers/CalendarHelper.cs

```diff
@@ -190,6 +221,27 @@ private static TagBuilder GetButtonContainerHtml(CalendarScheduledEventModel mod
             return button;
         }
 
+        private static string GetDayOfWeekClass(DayOfWeek dayOfWeek, CalendarDisplayType displayType)
+        {
+            string dayOfWeekClass = string.Empty;
+
+            switch (displayType)
+            {
+                case CalendarDisplayType.FullWeek:
+                    dayOfWeekClass = IsWeekend(dayOfWeek) ? "col-calendar-full-week weekend" : "col-calendar-full-week";
+                    break;
+
+                case CalendarDisplayType.WorkingWeek:
+                    dayOfWeekClass = IsWeekend(dayOfWeek) ? "d-none weekend" : "col-calendar-working-week";
```

Comment:
What about extracting text to private constants?

Reply:
I agree with you! Done.